flybywiresim/aircraft
# fix(a380x/EFB): updated checklists.json5 to match data from ECL

Fixes #[issue_no]

## Summary of Changes

Changed EFB checklist to match the data from the ECL. Also changed the oxygen checklist item to not be reversed when using the autofill checklist feature (eg. showing off on EFB when it is on and vice versa). 

## Screenshots (if necessary)
Before:
![image](https://github.com/user-attachments/assets/6b5e1fca-5a6a-4970-8580-f540dfd80bb4)

After:
![image](https://github.com/user-attachments/assets/85bd549f-fb35-44bc-9fc3-0ca7ddb80a53)

## References
N/A


## Additional context

Discord username (if different from GitHub): oim

## Testing instructions

1. Open checklist on the EFB and on the ECAM.
2. Verify that data matches between both lists when not checked and when checked. 
3. Use checklist at each stage of flight, verify that the checklist is properly marked as complete when marked at the correct stage (eg. the before start checklist should be marked green when the engines are off, and the other parts should not be able to be marked as complete).
4. Use the checklist with the autofill checklist feature enabled, ensure all working properly. 


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
